components/clipboard-button/index.js

```diff
@@ -68,7 +77,7 @@ class ClipboardButton extends Component {
 	render() {
 		// Disable reason: Exclude from spread props passed to Button
 		// eslint-disable-next-line no-unused-vars
-		const { className, children, onCopy, text, ...buttonProps } = this.props;
+		const { className, children, onCopy, onFinishCopy, text, ...buttonProps } = this.props;
```

Comment:
I'm not sure I understand this. :)

Reply:
Oh, removing it from `buttonProps`...

Reply:
I’m not a huge fan either, would’ve used `omit`.